src/appleseed/renderer/api/rendering.h

```diff
@@ -45,7 +45,6 @@
 #include "renderer/kernel/rendering/masterrenderer.h"
 #include "renderer/kernel/rendering/nulltilecallback.h"
 #include "renderer/kernel/rendering/progressive/progressiveframerenderer.h"
-#include "renderer/kernel/rendering/scenepicker.h"
```

Comment:
The fact that it's used in .studio makes removing this line problematic though. How can it compile?